pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -5,8 +5,8 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
 ---
 - authentication_required!
 - forbidden! unless dashboard_user[:user_type] == 'teacher'
-- error_string_none_selected = I18n.t('dashboard_error_none_selected')
-- error_string_other_section = I18n.t('dashboard_error_other_section')
+- error_string_none_selected = I18n.t('dashboard_error_none_selected').gsub!(/"/,"\\\\\"")
```

Comment:
I think you want use `gsub` not `gsub!` (see http://ruby-doc.org/core-2.1.4/String.html#method-i-gsub for the difference between them). Did you test this with strings with and without double quotes? ;)